### ISSUE TITLE
0.2.7 - save_json does not work?

### ISSUE THREAD

**Issue by Mansuete** [evidentlyai/evidently#538]
I have got an error "TypeError: Object of type Series is not JSON serializable" when trying to save json report for evidently.test_suite.test_suite.TestSuite object

**Comment by emeli-dral** [comment]
Hi @Mansuete,
thank for reporting this. We had significant Metrics refactoring in the last release, it looks like we left  pd.Series somewhere in MetricResult. 

Could you please share what tests did you use? It will help to locate a problem faster.

**Comment by Mansuete** [comment]
Hi @emeli-dral. Thanks for quick reply! Here is an example. Fail in report.save_json(f'report.json') row
```
from evidently.report import Report
from evidently.test_suite import TestSuite
from evidently.test_preset import BinaryClassificationTestPreset, DataStabilityTestPreset, DataQualityTestPreset
from evidently.metric_preset import ClassificationPreset, DataDriftPreset
from sklearn.datasets import load_breast_cancer
import numpy as np
import pandas as pd

dataset = load_breast_cancer(as_frame=True)


ref = pd.DataFrame(dataset['target'])
curr = pd.DataFrame(dataset['target'])


ref['prediction'] = [np.random.random() for i in range(ref.shape[0])]
curr['prediction'] = [np.random.random() for i in range(curr.shape[0])]

report = Report(metrics=[ClassificationPreset(probas_threshold=0.7)])

report.run(reference_data=ref, current_data=curr)

report.save_html(f'report.html')
report.save_json(f'report.json')

```

**Comment by emeli-dral** [comment]
Thanks a lot! It has helped!

We are testing quick fix in the #537

**Comment by emeli-dral** [comment]
@Mansuete fix is merged.

Now you can build evidently from the source code.
Here is a code snapshot to do so:

```
git clone https://github.com/evidentlyai/evidently.git evidently_repo

cd /path/to/evidently_repo

python3 -m venv venv
. venv/bin/activate

pip install -e .
```

Fixed version will be released soon as well.
